third_party/sycl/sycl/BUILD.tpl

```diff
@@ -10,16 +10,27 @@ package(default_visibility = ["//visibility:public"])
 exports_files(["LICENSE.text"])
 
 config_setting(
-    name = "using_sycl",
+    name = "using_sycl_ccpp",
     values = {
         "define": "using_sycl=true",
+        "define": "using_trisycl=false",
     },
 )
 
+config_setting(
+    name = "using_sycl_trisycl",
+    values = {
+        "define": "using_sycl=true",
+        "define": "using_trisycl=true",
```

Comment:
Looks like bazel is not happy with this here.
I will roll this change back and then we can look into modifying this config setting.